src/features/user/MFV2/ContributionsMap/Common/common.module.scss

```diff
@@ -1,7 +1,9 @@
+@import '../../../../.././theme/theme';
+
 .mapCreditsContainer {
   position: absolute;
   right: 51px;
-  bottom: 0;
+  bottom: -3px;
   margin-bottom: 10px;
```

Comment:
I'm not sure why `bottom: -3px` was added, but it doesn't seem to be needed.

Reply:
The bottom attribute is required for positioning the Map credit info Icon, instead of margin-bottom and z-index, which have been removed.

Reply:
The map credit icon still appears misaligned with the zoom controls. The bottom of the icon and the bottom of the zoom controls should coincide, but the map credit icon is currently a little lower.